Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/Helpers.java

```diff
@@ -110,6 +110,49 @@ public static int toInt(BigInteger i) {
         return i.intValue();
     }
 
+    public static void outOfRange(String msg) {
+          //System.out.println(msg);
+          //System.exit(-1);
+          throw new dafny.DafnyHaltException(msg);
+    }
+
+    public static int toIntChecked(BigInteger i, String msg) {
+        int r = i.intValue();
+        if (!BigInteger.valueOf(r).equals(i) || (msg != null && r < 0)) {
```

Comment:
I don't understand the second disjunct. When would it ever be used in a situation that's not already caught by (the static verifier and) the first disjunct?

Reply:
Obsolete code -- removed